node/tessel-export.js

```diff
@@ -555,6 +567,29 @@ Tessel.Pin.prototype.read = function(cb) {
   return this;
 };
 
+Tessel.Pin.prototype.mode = function(pullType, cb) {
```

Comment:
Ah, actually, I just misread the T1 firmware docs. Apparently, it's supposed to be called `.pull`. I'll go with that to keep backwards compat.